package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@xcoobee/payment-sdk",
+  "author": "Sergey S",
+  "contributors": [
+    "Bilal S",
+    "Dan H"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "version": "0.8.4",
+  "description": "",
+  "homepage": "https://github.com/XcooBee/payment-sdk-js#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/XcooBee/payment-sdk-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/XcooBee/payment-sdk-js/issues"
+  },
+  "files": [
+    "dist/"
+  ],
+  "main": "dist/node/node.js",
+  "module": "dist/node/node.js",
+  "browser": "dist/browser/browser.js",
+  "scripts": {
+    "test": "mocha -r ts-node/register tests/**/*.test.ts",
+    "coverage": "nyc -r lcov -e .ts -x \"*.test.ts\" npm run test",
+    "docs": "typedoc",
```

Comment:
We should clean the `docs` directory before we re-generate new docs so that old stuff gets removed.